### PR TITLE
Feature home

### PR DESCRIPTION
**Pull request by umiatiyah**
how about this page?
image

need your suggestion for structure code
image

### CODE REVIEW

File: src/views/home/index.tsx [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
 const Home = () => {
   return (
-    <div className="bg-red-500">
-      Test BG
-      <h1 className="text-3xl font-bold underline">Hello world!</h1>
-      <h1 className="">Hello world!</h1>
-    </div>
+    <>
+      <main className="bg-gray-900 text-white font-sans">
+        <Navbar />
```
Comment (fathrezza146) [inline src/views/home/index.tsx]: navbar, header, and footer can be placed at router

**Comment by fathrezza146** [comment]
each components in global or home should be wrapped in folders, in case you need any hooks for each section or need additional component inside.

For Layout please put it in components folder

**Review by fathrezza146 — CHANGES_REQUESTED** [review]

File: src/views/home/index.tsx (hunk 1 above)
Comment (umiatiyah) [reply src/views/home/index.tsx]: i've put it into router, please check

**Comment by umiatiyah** [comment]
> each components in global or home should be wrapped in folders, in case you need any hooks for each section or need additional component inside.
> 
> For Layout please put it in components folder

ok, i've replace layout into components folder.
but I don't understand how about components in home, should I wrap all section into one file tsx? then all section home (about, contact, portfolio) in one file index.tsx in home @fathrezza146

**Comment by fathrezza146** [comment]
> > each components in global or home should be wrapped in folders, in case you need any hooks for each section or need additional component inside.
> > For Layout please put it in components folder
> 
> ok, i've replace layout into components folder. but I don't understand how about components in home, should I wrap all section into one file tsx? then all section home (about, contact, portfolio) in one file index.tsx in home @fathrezza146

no, that is good. but each sectio (about, contact, porto) should each be a floder.

**Review by fathrezza146 — APPROVED** [review]
lets discuss about foldering later lmao. for now its good

**Pull request merged**